packages/gridproxy_client/src/builders/abstract_builder.ts

```diff
@@ -40,23 +40,35 @@ export abstract class AbstractBuilder<T> {
     }
   }
 
-  public async build(path: string): Promise<Response> {
+  public async build(path: string, timeout = 5000): Promise<Response> {
```

Comment:
I think 5 seconds is small as if the filters are applied the request might take more time, especially on mainnet

Reply:
Would you like to make it configurable?